e2e/playwright/test-utils.ts

```diff
@@ -13,11 +13,15 @@ import fsp from 'fs/promises'
 import pixelMatch from 'pixelmatch'
 import type { Protocol } from 'playwright-core/types/protocol'
 import { PNG } from 'pngjs'
+import dotenv from 'dotenv'
+
+const NODE_ENV = process.env.NODE_ENV || 'development'
+dotenv.config({ path: [`.env.${NODE_ENV}.local`, `.env.${NODE_ENV}`] })
+export const token = process.env.token || ''
```

Comment:
Not super pleased with this either, open to suggestions

Reply:
Does `dotenv.config` still prefer environment variables over files?